docs/host-list.rst

```diff
@@ -0,0 +1,81 @@
+.. source: https://gist.github.com/Twentysix26/cb4401c6e507782aa6698e9e470243ed
+
+.. _host-list:
+
+=============
+VPS providers
+=============
+
+.. note::
+    This doc is written for the :ref:`hosting section <getting-started-hosting>`
+    of the :ref:`getting started <getting-started>` guide. Please take a look
+    if you don't know how to host Red on a VPS.
+
+This is a list of the recommanded VPS providers.
+
+.. warning::
+    Please be aware that a Linux server is controlled through command line.
```

Comment:
"through command line." is not grammatically correct. "through a command line." would be better.